### ISSUE TITLE
Using "pm open foo" opens document rather than project

### ISSUE THREAD

**Issue by shadowspawn** [saurabhdaware/projectman#25]
I added a project called `accounts`. When I use `pm` without arguments I can select `accounts` from the menu and it opens the expected project. When I try:

```
pm open accounts
```

I get an editor with an empty document called `accounts`. The settings have

```
"commandToOpen": "code",
```

I am on macOS Mojave, node v10.16.3.

(Only just installed, and looking forward to trying projectman over next week. Thanks for sharing.)

**Comment by saurabhdaware** [comment]
Can you show full content of `pm edit`?

**Comment by shadowspawn** [comment]
I was checking the contents for anything I wanted to redact and... the `accounts` entry ends with `accounts/accounts`, that duplication looks wrong! I wondered if I added that one incorrectly. I'll investigate further and report back.

**Comment by saurabhdaware** [comment]
Did you do `pm add accounts` while adding project?

**Comment by shadowspawn** [comment]
I suspect I did. I probably thought I was providing the project name and short-cutting the prompt. 

I reproduced the problem I had when I used `pm add accounts` from inside the accounts directory, and things worked correctly when either I used `pm add` inside accounts, or used `pm add accounts` from the parent directory.

Not sure if problem could reasonably have been detected when I made the mistake, but I am happy that it was a usage error, will close issue.

Thanks.
